src/ILCompiler.Compiler/src/Compiler/DependencyAnalysis/ConstructedEETypeNode.cs

```diff
@@ -137,11 +137,11 @@ protected override DependencyList ComputeNonRelocationBasedDependencies(NodeFact
 
             if (closestDefType.HasGenericDictionarySlot())
```

Comment:
Ok that seems to pass tests and didn't regress binary size so it's probably good.